Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["web-programming", "web-programming::http-server"]
 rust-version = "1.59"
 
 [features]
-default = ["binary"]
+default = []
 binary = [
     "anyhow", 
```

Comment:
Does this library use anyhow?

Reply:
Only for the binary and tests

Reply:
> Only for the binary and tests

Better to not use anyhow.
Is it a lot work to remove anyhow in binary?

Reply:
I think it's ok to use `anyhow`  for convenience in tests and the main function. Is there any better replacement?

Reply:
> I think it's ok to use `anyhow` for convenience in tests and the main function. Is there any better replacement?

So you mean anyhow is only used in main function and test? That's fine.

Reply:
A quick search
https://github.com/datenlord/s3-server/search?q=anyhow&type=